tmt/steps/report/polarion.py

```diff
@@ -35,77 +36,106 @@ class ReportPolarionData(tmt.steps.report.ReportStepData):
         default=None,
         option='--project-id',
         metavar='ID',
-        help='Use specific Polarion project ID.'
+        help=(
+            'Use specific Polarion project ID.'
+            'also uses environment variable TMT_PLUGIN_REPORT_POLARION_PROJECT_ID.')
```

Comment:
Missing leading space in this string, plus a dot finishing the sentence in the previous string => `... project ID.also ...`